pkg/ipc/ipc.go

```diff
@@ -40,6 +40,7 @@ const (
 	FlagEnableCgroups                            // setup cgroups for testing
 	FlagEnableCloseFds                           // close fds after each program
 	FlagEnableDevlinkPCI                         // setup devlink PCI device
+	FlagEnableNicVF                              // setup NIC VF device
```

Comment:
These consts are assigned automatically depending on their position in the list. Adding an item somewhere inside the list would require us to also change const values for all the following constants.

```
	flag_vhci_injection = flags & (1 << 12);
	flag_wifi = flags & (1 << 13);
	flag_delay_kcov_mmap = flags & (1 << 14);
```

Would have to become
```
	flag_vhci_injection = flags & (1 << 13);
	flag_wifi = flags & (1 << 14);
	flag_delay_kcov_mmap = flags & (1 << 15);
```

And your new flag would be 1<<12 in this case.

Please move it to the back of the list, then we won't need so many changes.

Reply:
Requested change made - see: https://github.com/gkennedy12/syzkaller/tree/pull_072622_1